test/e2e/helpers/visu-utils.ts

```diff
@@ -177,12 +177,11 @@ export class PageTester {
     expect(response.status()).toBe(200);
 
     const waitForSelectorOptions = { timeout: 5_000 };
-    const bpmnContainerElementHandle = await page.waitForSelector(`#${this.bpmnContainerId}`, waitForSelectorOptions);
     await expect(page.title()).resolves.toMatch(this.expectedPageTitle);
 
-    await page.waitForSelector(new BpmnQuerySelectors(this.bpmnContainerId).existingElement(), waitForSelectorOptions);
-
-    return bpmnContainerElementHandle;
+    const bpmnContainerId = 'bpmn-container';
+    await page.waitForSelector(new BpmnQuerySelectors(bpmnContainerId).existingElement(), waitForSelectorOptions);
+    return await page.waitForSelector(`#${bpmnContainerId}`, waitForSelectorOptions);
```

Comment:
⚠️ there is a logic change here that is not wanted IMO
Previously, we first ensured that the bpmn-container was available, if so, then we ensured that at least one bpmn element was available (so diagram was rendered).
Now, the order is inverted and we won't detect issue with the container: the bpmn element won't be find and we will think there is a rendering issue whereas the issue is on the container itself

Reply:
Changed